node/config/def.go

```diff
@@ -14,6 +14,14 @@ import (
 	sectorstorage "github.com/filecoin-project/lotus/extern/sector-storage"
 )
 
+const (
+	// DefaultRetrievalPricing configures the node to use the default retrieval pricing policy.
+	DefaultRetrievalPricing = "default"
+	// ExternalRetrievalPricing configures the node to use the external retrieval pricing script
+	// configured by the user.
+	ExternalRetrievalPricing = "external"
+)
```

Comment:
ubernit: invert the naming here, as doing so groups these consts alphabetically in godocs.

```suggestion
const (
	// RetrievalPricingDefault configures the node to use the default retrieval pricing policy.
	RetrievalPricingDefault = "default"
	// RetrievalPricingExternal configures the node to use the external retrieval pricing script
	// configured by the user.
	RetrievalPricingExternal = "external"
)
```

Reply:
This is done.